frontend/src/components/Skew/index.jsx

```diff
@@ -33,8 +33,40 @@ class Skew extends React.Component {
         this.props.getLabelCounts();
     }
 
+    renderTable(){
+        const { unlabeled_data, skewLabel, labels } = this.props;
+        if (unlabeled_data.length == 0) {
+            return <b>Loading unlabeled data. This may take up to a minute for large datasets..</b>;
```

Comment:
So it is trying to load all of the data. I could see pagination being helpful, though I don't know how simple it would be to use the search part of the table with it. I suppose the search functionality could be moved to a separate thing? It seems pretty important for someone to search for content that is related to a particular label.

Reply:
I added backend pagination, sorting, and filtering for the skew page. I'm guessing this is going to need to be added for the History page as well (since it will bog down once the number of labeled articles gets large).

Reply:
Agreed. Do you want to add that as part of this mr or just add it to the backlog?

Reply:
I'll add it to the backlog. I'd rather get these changes nailed down first.